designsystem/src/main/java/com/b1nd/dodam/designsystem/component/Card.kt

```diff
@@ -0,0 +1,260 @@
+import com.b1nd.dodam.designsystem.theme.DodamTheme
+
+@Composable
+fun DodamAskCard(
+    askStatus: AskStatus,
+    labelText: String? = null,
+    startTime: String = "",
+    startTimeText: String = "",
+    endTime: String = "",
+    endTimeText: String = "",
+    currentLeftTime: String = "",
+    reason: String = "",
+    rejectedReason: String? = null,
+    phoneReason: String? = null,
+    progress: Float = 0.0f,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .clip(shape = RoundedCornerShape(18.dp))
+            .background(MaterialTheme.colorScheme.background)
+            .padding(16.dp),
+    ) {
+        val (statusColor, statusText) = when (askStatus) {
+            AskStatus.ALLOWED -> MaterialTheme.colorScheme.primary to "승인됨"
+            AskStatus.PENDING -> MaterialTheme.colorScheme.tertiary to "대기중"
+            AskStatus.REJECTED -> MaterialTheme.colorScheme.error to "거절됨"
+        }
+        Row(verticalAlignment = Alignment.CenterVertically) {
+            Box(
+                modifier = Modifier
+                    .clip(shape = RoundedCornerShape(100))
+                    .background(statusColor)
+                    .padding(
+                        vertical = 4.dp,
+                        horizontal = 12.dp,
+                    ),
+            ) {
+                Text(
+                    text = statusText,
+                    style = MaterialTheme.typography.titleSmall,
+                    color = MaterialTheme.colorScheme.background,
+                )
+            }
+            Spacer(modifier = Modifier.weight(1f))
+            labelText?.let {
+                Text(
+                    text = labelText,
+                    style = MaterialTheme.typography.labelLarge,
+                    color = MaterialTheme.colorScheme.tertiary,
+                )
+            }
+        }
+        Spacer(modifier = Modifier.height(12.dp))
+        Text(
+            text = reason,
+            style = MaterialTheme.typography.bodyMedium,
+            color = MaterialTheme.colorScheme.onBackground,
+        )
+
+        if (askStatus != AskStatus.REJECTED) { // 신청이 승인, 대기중일 때
+            Spacer(modifier = Modifier.height(12.dp))
+            Divider(color = MaterialTheme.colorScheme.secondary)
+            Spacer(modifier = Modifier.height(12.dp))
+            Column {
+                Row {
+                    Column(
+                        modifier = Modifier
+                            .weight(1f)
+                            .padding(end = 16.dp)
+                            .align(Alignment.Bottom),
+                    ) {
+                        Row(verticalAlignment = Alignment.Bottom) {
+                            Text(
+                                text = currentLeftTime,
+                                style = MaterialTheme.typography.titleSmall,
+                                color = MaterialTheme.colorScheme.onBackground,
```

Comment:
Please change the background color to MaterialTheme.colorScheme.onSurfaceVariant